paasta_tools/cli/cmds/local_run.py

```diff
@@ -83,6 +83,9 @@ def perform_http_healthcheck(url, timeout):
             "Multiple content-type headers detected in response."
             " The Mesos healthcheck system will treat this as a failure!"))
         return (False, "http request succeeded, code %d" % res.status_code)
+    # check if there is response content in the body
+    if len(res.content) < 1:
```

Comment:
`if not res.content` is the PEP 8-compliant way to check if a sequence is empty.